include/capstone/riscv.h

```diff
@@ -53,6 +53,8 @@ typedef struct cs_riscv {
 	// Number of operands of this instruction, 
 	// or 0 when instruction has no operand.
 	uint8_t op_count;
+	// Does this instruction need effective address or not.
+	uint8_t need_effective_addr;
```

Comment:
we can use `bool` type here instead of `uint8_t`?

Reply:
now you add a new field to cs_riscv, be sure to update Python binding, too.

also, better to put `op_count` next to `operands[]`, but not separate by this field `need_effective_addr`.